doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
-- **Container Image**:	Stored instance of a container that holds a set of software needed to run an application.
-- **Container**:	A lightweight and portable executable image that contains software and all of its dependencies.
+- **Container:**	A lightweight and portable executable image that contains software and all of its dependencies.
 >_*Note:*_ OCI defines **Container** as "An environment for executing processes with configurable isolation and resource limitations. For example, namespaces, resource limits, and mounts are all part of the container environment."
 A **Container** provides operating-system-level virtualisation by abstracting the “user space”. One big difference between **Containers** and **VMs** is that unlike VMs, where each **VM** is self-contained with all the operating systems components are within the **VM** package, containers "share" the host system’s kernel with other containers.
 
-- **Container Runtime**: The software that is responsible for running containers.
->_*Note:*_ as explained in [OCI Glossary](https://github.com/opencontainers/runtime-spec/blob/master/glossary.md) it reads the configuration files for a **Container** from a directory structure, uses that information to create a container, launches a process inside the container, and performs other lifecycle actions.
+- **Container Engine:** Software components used to create, destroy, and manage containers on top of an operating system.
 
-- **Container Engine**: Software components used to create, destroy, and manage containers on top of an operating system.
+- **Container Image:**	Stored instance of a container that holds a set of software needed to run an application.
 
-- **Pod**:	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
-- **Kubernetes Cluster**: A set of machines, called nodes and master, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
+- **Container Runtime:** The software that is responsible for running containers.
+>_*Note:*_ as explained in [OCI Glossary](https://github.com/opencontainers/runtime-spec/blob/master/glossary.md) it reads the configuration files for a
+**Container** from a directory structure, uses that information to create a container, launches a process inside the container, and performs other lifecycle actions.
+
+- **Kubernetes Cluster:** A set of machines, called nodes and master, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
```

Comment:
what is a machine -- not defined?

Reply:
Should we correct "machine" to "server" in another PR?

Reply:
@pgoyal01, can we resolve this thread?

Reply:
@EditKoselak Please resolve if addressed in another PR

Reply:
Thank you:) We opened #1970, so I'll resolve this discussion:)